atomate/vasp/firetasks/electrode_tasks.py

```diff
@@ -76,11 +77,11 @@ class AnalyzeChgcar(FiretaskBase):
     - "ChargeInsertionAnalyzer_kwargs": The kwargs to overwrite default values in the ChargeInsertionAnalyzer
     - "volumetric_data_type": the type of Charge density file to be used for analysis "CHGCAR"/"AECCAR"
     fw_spec optional:
-    - "max_insertions": Restrict the number of insertion sites based on charge density (default 5)
+    - "attempt_insertions": Restrict the number of insertion sites based on charge density (default 5)
     """
 
     def run_task(self, fw_spec):
-        max_insertions = fw_spec.get("max_insertions", 5)
+        max_insertions = fw_spec.get("attempt_insertions", 5)
```

Comment:
What do you think about making the default here the same as the default in `atomate/vasp/workflows/base/electrode.py`?

Reply:
ooops yeah definitely.

Reply:
Changed and ready for merge thanks!